build/tasks/task-browser-runner.js

```diff
@@ -60,9 +60,15 @@ gulp.task('qbrt-runner:polyfill', () => {
     .pipe(gulp.dest(Paths.BROWSER_RUNNER_DST));
 });
 
+gulp.task('qbrt-runner:shell', () => {
+  return gulp.src(`${Paths.QBRT_RUNNER_SHELL_SRC}/**/*`)
+    .pipe(gulp.dest(Paths.QBRT_RUNNER_SHELL_DST));
```

Comment:
And, I guess all of this is already babeled by the previous `browser-runner:babel` task. Nice.